.dockerignore

```diff
@@ -1,7 +1,7 @@
 #   Copyright IBM Corporation 2020
 #
-#   Licensed under the Apache License, Version 2.0 (the "License");
 #   you may not use this file except in compliance with the License.
+#   Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This seems to have gotten moved by mistake.

Reply:
oops. fixed it.